packages/webapp/src/pages/Integration/AuthSettings.tsx

```diff
@@ -232,9 +240,9 @@ export default function AuthSettings(props: AuthSettingsProps) {
                             )}
                         </div>
                     )}
-                    {showEditIntegrationIdMenu && integrationIdEdit && (
+                    {isTyping && integrationIdEdit && (
                         <div className="flex items-center border border-border-gray bg-active-gray text-white rounded-md px-3 py-0.5 mt-0.5 cursor-pointer">
-                            <PencilSquareIcon className="flex h-5 w-5 cursor-pointer hover:text-zinc-400" onClick={() => editIntegrationID()} />
+                            <CheckIcon className="flex h-5 w-5 cursor-pointer hover:text-zinc-400" onClick={() => onSaveIntegrationID()} />
```

Comment:
Let's keep the icon the same

Reply:
Thank you for that, I have just reverted back to the previous icon.